llvm/lib/Target/ARM/ARM.td

```diff
@@ -1747,4 +1747,5 @@ def ARM : Target {
   let AssemblyParsers = [ARMAsmParser];
   let AssemblyParserVariants = [ARMAsmParserVariant];
   let AllowRegisterRenaming = 1;
+  let PreferSmallerInstructions= true;
```

Comment:
Should it be a member of AsmParser instead?

Reply:
Ah yeah that would make sense, I am happy to move it.

Reply:
Will continue this in https://github.com/llvm/llvm-project/pull/83587